includes/api/Cookies_Settings_Shortcode.php

```diff
@@ -0,0 +1,80 @@
+<?php
+// Shorcode which enables the form
+// ont the Cookies Settings page
+function shortcode_settings_page( $atts ) {
+    $a = shortcode_atts( array(
+    'usage' => 'show',
+    'settings' => 'hide',
+    'necessary' => 'show',
+    ), $atts );
+    
+    $accept_usage_option = "";
+    $accept_settings_option = "";
+    $reject_usage_option = "";
+    $reject_settings_option = "";
+
+    $cookies_policy_to_obj = decode_cookie('cookies_policy');
+
+    // If the user accept the Google Analitics cookies
+    // turn ON the radio selection
+    if ($cookies_policy_to_obj->usage === true) {
```

Comment:
Does this test need to be wrapped in a test to ensure the `$cookies_policy_to_obj` object has a `usage` property. Something like `property_exists()` (https://www.php.net/manual/en/function.property-exists.php). Or you could update `decode_cookie` to always ensure the relevant properties are always present, regardless of whether they were included in the JSON.

This would apply for all tests for a given property.

What do you think?